features/step_definitions/common_steps.rb

```diff
@@ -176,9 +177,8 @@
 #       i.e. either cassette must exist, or you explicitly call "and record 'none'"
 #
 And(/^I insert the VCR cassette '(.*?)'(?: and record '(.*?)')?$/) do |name, record|
-  # Caching.clear if record
   record_mode = ENV['FEE_CALC_VCR_MODE']&.to_sym if fee_calc_vcr_tag?
-  record_mode ||= (record || 'once').to_sym
+  record_mode ||= (record || 'none').to_sym
```

Comment:
could change this back to `once` (which is vcr default)

vcr [once mode](https://relishapp.com/vcr/vcr/v/3-0-3/docs/record-modes/once)

Reply:
but i quite like complete blocking of unrecorded interactions 🤔